projects/angular/src/data/tree-view/tree-node.ts

```diff
@@ -118,6 +119,14 @@ export class ClrTreeNode<T> implements OnInit, AfterContentInit, AfterViewInit,
     this._model.nodeId = this.nodeId;
   }
 
+  @Input('clrDisabled')
+  get disabled(): boolean {
+    return this._model.disabled;
+  }
+  set disabled(value: boolean) {
+    this._model.disabled = value || this._model.parent?.disabled;
```

Comment:
```suggestion
    this._model.disabled = value;
```

I don't think we need this anymore. We should write the state as given. The parent state is accounted for when read.

Reply:
I thought I removed that.